### ISSUE TITLE
Veil all user/bidder attributes

### ISSUE THREAD

**Issue by adelevie** [18F/micropurchase#246]
Right now, we veil bidder attributes ad-hoc
e.g just the name, just the duns
since those are the only attrs needed in the /auctions/:id/bids view
but for the API, all data must be presented
so we really should have a `veiled_bidder` method which returns a hash of something like:
```ruby
{
  id: 'Veiled until auction ends',
  # this ^, but for all attributes
}
```

**Comment by harrisj** [comment]
Fixed in PR #308 I think?

**Comment by jessieay** [comment]
this is done. closing
